api/API-CONTRACT.md

```diff
@@ -0,0 +1,138 @@
+# BBSTrader API Contract
+
+## Objects
+
+### Account object
+```
+{
+  login: integer
+  balance: float
+  profit: float
+  equity: float
+  server: string
+}
+```
+
+### ClosedOrder object
+```
+{
+  ticket: integer
+  time: milliseconds since epoch
+  order_type: integer
+  volume: float
+  profit: float
+  symbol: string
+  comment: string
+}
+```
+
+### OpenOrder object
+```
+{
+    ticket: integer
+    time_setup: milliseconds since epoch
+    order_type: integer
+    volume_current: float
+    stop_loss: float
+    take_profit: float
+    price_open: float
+    price_current: float
+    symbol: string
+    comment: string
```

Comment:
Our users don't add these comments themselves to the orders and so this field is not meaningful to them. Let's go ahead and take it out of our return responses.

Reply:
Ok